src/Connection.php

```diff
@@ -268,6 +263,16 @@ public function connection()
         return $this->connection;
     }
 
+    /**
+     * Returns the default driver type set for the connection in $driverType.
+     *
+     * @return string|null
+     */
+    public static function defaultDriverType()
+    {
+        return (new \ReflectionClass(static::class))->getDefaultProperties()['driverType'] ?? null;
```

Comment:
could you please make sure that the changes work with atk4/schema ?

Reply:
this only duplicates registry defaults, I this we should not have this static method at all...